docs/getting-started.stories.mdx

```diff
@@ -236,41 +238,51 @@ Provided `aegeanTheme`:
 <CarbonProvider theme={aegeanTheme}>Children</CarbonProvider>
 ```
 
-#### AppWrapper
-
-This component wraps all components within the width constrains of your application.
-
-```jsx
-<AppWrapper>Children</AppWrapper>
-```
-
-You should refer to our [Storybook documentation](https://carbon.sage.com/?path=/docs/app-wrapper--default) for details.
-
 #### Quickstart
 
 A basic project `index.js` file would resemble this example.
 
 ```jsx
 import React from "react";
 import ReactDOM from "react-dom";
-import { ThemeProvider } from "styled-components";
-import mintTheme from "carbon-react/lib/style/themes/mint";
-import "carbon-react/lib/utils/css";
-import AppWrapper from "carbon-react/lib/components/app-wrapper";
+import useMediaQuery from "carbon-react/lib/hooks/useMediaQuery";
+import CarbonProvider from "carbon-react/lib/components/carbon-provider";
+import GlobalStyle from "carbon-react/lib/style/global-style";
+import Box from "carbon-react/lib/components/box";
 import Button from "carbon-react/lib/components/button";
 
-const App = (props) => {
+const App = () => {
+  const isBelowLargeScreen = useMediaQuery("(max-width: 1024px)");
+  const isBelowWideScreen = useMediaQuery("(max-width: 1366px)");
+  const createPadding = () => {
+    if (isBelowLargeScreen) return "0 25px";
+    if (isBelowWideScreen) return "0 30px";
+    return "0 40px";
+  };
+  const boxProps = {
+    boxSizing: "border-box",
+    "max-width": "1600px",
+    "min-width": "958px",
+    margin: "0 auto",
+    padding: createPadding(),
+  };
+
   return (
-    <ThemeProvider theme={mintTheme}>
-      <AppWrapper>
+    <CarbonProvider>
+      <GlobalStyle />
+      <Box {...boxProps}>
         <Button>Hello Carbon</Button>
         <p>Please remember to select the appropriate version of Carbon.</p>
-      </AppWrapper>
-    </ThemeProvider>
+      </Box>
+    </CarbonProvider>
   );
 };
 
 ReactDOM.render(<App />, document.getElementById("app"));
 ```
 
-This can also be found in our [Codesandbox template](https://codesandbox.io/s/carbon-quickstart-xi5jc).
+This can also be found in our [Codesandbox template](https://codesandbox.io/s/carbon-quickstart-j5pb2).
+
+#### Deprecation of AppWrapper
```

Comment:
Hi @Parsium - this doesn't really belong in the getting started docs, I think you should put this in the `development-roadmap` docs in the `Deprecations` section